website/docs/r/edgegateway_settings.html.markdown

```diff
@@ -0,0 +1,94 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_edgegateway_settings"
+sidebar_current: "docs-vcd-resource-edgegateway-settings"
+description: |-
+  Provides a vCloud Director edge gateway global settings. This can be used to update global edge gateways settings related to firewall and load balancing.
+---
+
+# vcd\_edgegateway\_settings
+
+Provides a resource that can update vCloud Director edge gateway global settings either as system administrator or as
+organization user.
+
+The main use case of this resource is to allow both providers and tenants to change edge gateways global settings (such as
+enabling load balancing or firewall) when the edge gateway was created outside of terraform.
+
+~> **Warning:** The edge gateway settings info is tied to an edge gateway. Thus, there could be only one instance per 
+edge gateway. Using a different definition for the same edge gateway ID will result in a previous instance to be overwritten.
```

Comment:
What happens if there are two org admins editing global settings of the same EG? Should we put a warning about that as well?

Reply:
I don't understand. This warning is exactly about that.

Reply:
What I meant is that now we say "there could be only one instance per edge gateway", but when you read this from a tenant user perspective you think it's about an instance in your .tf file.
However, we have a wider issue here, that there may be totally different tenant users trying to change the same EG with global settings each in their own .tf configs without knowing that.
So, I was just thinking that maybe we should explicitly state this unwanted scenario.

p.s. I agree that current description technically implies that already, it's just may be missed as it's not explicit.

Reply:
I could add the clarification, although the risk is unlikely:

* Each EGW belongs to one tenant;
* The main use case for this resource calls for it to be used once per EGW;
* The changes required are usually of type "enable global LB so that I can set up my own LB". Thus, even if this change is set at the start of two scripts that use LB, they are not creating conflicting settings. The possibility of tenants shooting themselves in the foot are really slim.

Reply:
Yeah, I was thinking of when there's two Org admins, who don't talk with each other, using the same EG. OK, let's assume current description is enough.